tool/jt.rb

```diff
@@ -148,11 +162,12 @@ def help
     puts 'jt irb                                         irb'
     puts 'jt rebuild                                     clean and build'
     puts 'jt run [options] args...                       run JRuby with -X+T and args'
-    puts '    --graal        use Graal (set GRAAL_BIN or it will try to automagically find it)'
-    puts '    --asm          show assembly (implies --graal)'
-    puts '    --server       run an instrumentation server on port 8080'
-    puts '    --igv          make sure IGV is running and dump Graal graphs after partial escape (implies --graal)'
-    puts '    --jdebug       run a JDWP debug server on 8000'
+    puts '    --graal         use Graal (set GRAAL_BIN or it will try to automagically find it)'
+    puts '    --asm           show assembly (implies --graal)'
+    puts '    --server        run an instrumentation server on port 8080'
+    puts '    --igv           make sure IGV is running and dump Graal graphs after partial escape (implies --graal)'
+    puts '    --[j]debug      run a JDWP debug server on 8000'
+    puts '    --jexception[s] print java exceptions'
```

Comment:
It seem's to me that none of those (singular or plural) is a clear choice so I've added both. I can certainly use just one, which one would you choose?